court_scraper/base/selenium_base.py

```diff
@@ -0,0 +1,20 @@
+from selenium.webdriver.common.keys import Keys
+
+class SeleniumPage:
```

Comment:
class and module names should generally be aligned. So the class should be `SeleniumBase` or the module should be `base/selenium_page.py`